build.sh

```diff
@@ -28,7 +28,7 @@ fi
 
 # This installs the latest build of the dotnet targetting pack net461 via NuGet for linux desktop clr builds.
 # The package source can be removed once Microsoft.TargetingPack.NETFramework.v4.6.1 comes out of alpha on NuGet.
-dotnet restore ./buildscripts/BuildScripts.csproj -s https://dotnet.myget.org/F/dotnet-core/api/v3/index.json
+dotnet restore ./buildscripts/BuildScripts.csproj
```

Comment:
(Specifying the package source here is no longer required, since the preceding `nuget restore Castle.Core.sln` (run by Travis CI) already downloaded the package from the alternate NuGet feed.)